recipes/cccl/all/test_package/conanfile.py

```diff
@@ -18,7 +18,7 @@ def build(self):
         with tools.environment_append(environment):
             cxx = tools.get_env("CXX")
             self.run("{cxx} {src} -o example".format(
-                cxx=cxx, src=os.path.join(self.source_folder, "example.cpp")), win_bash=self.settings.os is "Windows")
+                cxx=cxx, src=os.path.join(self.source_folder, "example.cpp")), win_bash=self.settings.os == "Windows")
```

Comment:
Same fix here; '==' avoids potential identity pitfalls. Consider whether case sensitivity matters for Windows checks.